frontend/components/brand/products/product-gallery.tsx

```
@@ -0,0 +1,71 @@
+  return (
+    <div className="flex flex-col-reverse gap-4 lg:grid lg:grid-cols-[100px_1fr] lg:gap-6">
+      <div className="flex gap-4 overflow-auto px-1 lg:flex-col">
+        {images.map((image, index) => (
+          <button
+            key={index}
+            onClick={() => setCurrentImage(index)}
+            className={cn(
+              "relative aspect-square h-[100px] overflow-hidden rounded-lg border-2",
+              currentImage === index ? "border-primary" : "border-transparent",
+            )}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)